codegenerator/cli/templates/dynamic/codegen/src/db/Entities.res.hbs

```diff
@@ -23,257 +24,6 @@ let isIndex = true
 @genType
 type whereOperations<'entity, 'fieldType> = {eq: 'fieldType => promise<array<'entity>>}
 
-module EntityHistory = {
-  type historyFieldsGeneral<'a> = {
```

Comment:
Moved to its own module, there are also a number of changes so might be nicer to review diffs of commits after the move.